login-workflow/README.md

```diff
@@ -18,7 +18,7 @@ This package is flexible, allowing you to use the Login and Registration flows i
 
 These workflows are back-end agnostic, meaning you can use them with any back-end API you wish. You simply need to provide an implementation for several key functions (actions) that are called at various points within the workflows based on user interaction.
 
-![Login](https://raw.githubusercontent.com/etn-ccis/blui-react-workflows/master/login-workflow/media/login.png)
+![Login](https://raw.githubusercontent.com/etn-ccis/blui-react-workflows/master/login-workflow/media/okta-login.png)
```

Comment:
@surajeaton , Do we need to add [Okta Login] here?

Reply:
Yes okta login is the recommended login screen, it is better to keep okta login screen shot there

Reply:
> Okta Login



> Yes okta login is the recommended login screen, it is better to keep okta login screen shot there

Yes, can we rename Login to Okta Login
```suggestion
![Okta Login](https://raw.githubusercontent.com/etn-ccis/blui-react-workflows/master/login-workflow/media/okta-login.png)
```